deployment/kubernetes/populate_db_job.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: populate-db-job
+spec:
+  ttlSecondsAfterFinished: 120 # the job pod will auto-delete 120 secs after completion
+  template:
+    spec:
+      containers:
+      - name: populate-db-job
+        image: pdc/populate_db:1.0
+        imagePullPolicy: Never
+        env:
+          - name: DB_HOST
+            valueFrom:
+              configMapKeyRef:
+                name: config-dev
+                key: DB_HOST
+          - name: DB_PORT
+            valueFrom:
+              configMapKeyRef:
+                name: config-dev
+                key: DB_PORT
+          - name: DB_USERNAME
+            valueFrom:
+              configMapKeyRef:
+                name: config-dev
+                key: DB_USERNAME
+          - name: DB_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: secret-dev
+                key: DB_PASSWORD
+          - name: DB_NAME
+            valueFrom:
+              configMapKeyRef:
+                name: config-dev
+                key: DB_NAME
+          - name: SUPABASE_KONG_HOST
+            valueFrom:
+              configMapKeyRef:
+                name: config-dev
+                key: SUPABASE_KONG_HOST
+          - name: SUPABASE_KONG_PORT
+            valueFrom:
+              configMapKeyRef:
+                name: config-dev
+                key: SUPABASE_KONG_PORT
```

Comment:
```suggestion
          - name: SUPABASE_KONG_PORT
            value: "8000"
```

Reply:
@lsetiawan 
Just to confirm, for these cases, where the values are not pre-existing in the cluster configmap, do you want me to hard-code those in the `job.yaml`?


Yesterday, I believe we were more inclined towards passing these values through the `kubectl` command line like below for flexibility, right?

```
kubectl create configmap config-dev \
  --from-literal=DB_HOST="supabase-supabase-db" \
  --from-literal=DB_PORT="5432" \
  --from-literal=SUPABASE_KONG_HOST="supabase-supabase-kong" \
  --from-literal=SUPABASE_KONG_PORT="8000"
```